<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QMainWindow" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1">
     <widget class="QPushButton" name="ButtonAtan">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Xmax =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Button2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="ButtonPow">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="ButtonSqrt">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Button6">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Button8">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="Button3">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Button5">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="xVal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="ButtonDot">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="ButtonSin">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Button9">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Xmin =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="ButtonTan">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="ButtonCloseBr">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="ButtonAcos">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="ButtonSub">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="ButtonAdd">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button7">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="ButtonMul">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="ButtonCos">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="ButtonClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #4B0082;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="ButtonMod">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="ButtonOpenBr">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="Button1">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="xMax">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="5" rowspan="3">
     <widget class="QPushButton" name="ButtonEqual">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #000000;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QPushButton" name="Button0">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="ButtonAsin">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="ButtonDel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="ButtonDiv">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button4">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="DisplayX">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>X =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="xMin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="ButtonX">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #D2691E;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3" colspan="3">
     <widget class="QPushButton" name="Button0_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #696969;
	border: 1px solid #696969;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #DCDCDC;
	border: 1px solid #696969;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>graph</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
